apps/web/src/app/[locale]/settings/page.tsx

```diff
@@ -32,7 +34,9 @@ export default async function SettingsPage() {
 
   return (
     <>
-      <Settings user={profileData} />
+      <Suspense fallback={<SettingsSkeleton />}>
```

Comment:
This suspense boundary will not be triggered, unless we move the profileData fetch there

Reply:
Hey man, can you please explain this better? 
I have tested it by delaying the rendering of the component, and it is triggered.

Reply:
i’ll pull your branch and verify/look deeper at the code shortly

Reply:
Wasn't the `loading.tsx` file used in that case? Was the entire page a skeleton or just this section?

Reply:
The `loading.tsx` file was used. 
I also removed the skeleton in that section in my most recent push. 

I think i get what you mean now, so i removed the suspense for just that section. It now relies solely on `loading.tsx`